dark1/memberavatarstatus/adm/style/acp_mas_main.html

```diff
@@ -99,7 +99,112 @@ <h2>{{ lang('ACP_MAS_MODE_MAIN') }}</h2>
 						</dd>
 					</dl>
 				</fieldset>
-			</td> 
+			</td>
+			<td rowspan="2">
+				<fieldset>
+					<legend>{{ lang('ACP_MAS_MODE_SEARCH') }}</legend>
+						<dl>
+							<dt>
+								<label>{{ lang('ACP_MAS_SH_FP_AV_LB') }}{{ lang('COLON') }}</label>
+							</dt>
+							<dd>
+								<div id="dark1_mas_sh_fp_av" class="toggleWrapper {% if MAS_SH_FP_AVATAR %}toggleChecked{% endif %}" >
+									<span for="dark1_mas_sh_fp_av" class="toggle">
+										<span class="toggle_handle"></span>
+									</span>
+								</div>
+							</dd>
+						</dl>
+						<dl>
+							<dt>
+								<label>{{ lang('ACP_MAS_SH_FP_AV_SZ_LB') }}{{ lang('COLON') }}</label>
```

Comment:
`{{ lang('ACP_MAS_SH_FP_AV_SZ_LB') ~ lang('COLON') }}` is better concatenated.
Same applies for all of the occurencies. from now on.

Reply:
Done 👍